src/main/resources/modules/general_anesthesia.json

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
Yeah, it seems a little weird to put it in the `medications` module since it includes a procedure (intubation).  Do you think an `anesthesia` directory makes sense, or would it help to make it more generic (e..g, `surgical_prep`)?

Reply:
Done - this is now in the `surgery` folder.